src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+.header {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
this property is redundant, delete it
for what reason did you add this property? probably you don't need it